src/domain/bitcoin/onchain/index.types.d.ts

```diff
@@ -44,11 +44,21 @@ type TxFilter = {
   apply(txsonChainTransactions: SubmittedTransaction[]): SubmittedTransaction[]
 }
 
+type FindOnChainFeeArgs = {
+  txHash: OnChainTxHash
+  scanDepth: number
+}
+
 interface IOnChainService {
-  getIncomingTransactions(
+  listIncomingTransactions(
     scanDepth: number,
   ): Promise<SubmittedTransaction[] | OnChainServiceError>
 
+  findOnChainFee({
```

Comment:
`find` seems weird as a prefix here.
Find implies it could be there - or not (like `repo.findById`)... here its more a `lookup` or something like that.

Reply:
changed